qt_gui_cpp/src/qt_gui_cpp/settings.cpp

```diff
@@ -34,70 +34,76 @@
 
 #include <stdexcept>
 
-namespace qt_gui_cpp {
+namespace qt_gui_cpp
+{
 
-Settings::Settings(QObject* obj)
-  : proxy_(obj)
+Settings::Settings(QObject * obj)
+: proxy_(obj)
 {}
 
-Settings Settings::getSettings(const QString& prefix)
+Settings Settings::getSettings(const QString & prefix)
 {
   Settings settings(proxy_.proxiedObject());
-  bool rc = proxy_.invokeMethodWithReturn("get_settings", Q_RETURN_ARG(Settings, settings), Q_ARG(QString, prefix));
-  if (!rc) throw std::runtime_error("Settings::get_settings() invoke method failed");
+  bool rc = proxy_.invokeMethodWithReturn("get_settings", Q_RETURN_ARG(Settings, settings),
+      Q_ARG(QString, prefix));
+  if (!rc) {throw std::runtime_error("Settings::get_settings() invoke method failed");}
```

Comment:
Since we are updating this anyway, I'd rather this conform to our usual style:

```suggestion
  if (!rc) {
    throw std::runtime_error("Settings::get_settings() invoke method failed");
  }
```

(same goes for all of the other ones below)

Reply:
https://github.com/ros-visualization/qt_gui_core/pull/295/commits/682f4e289e8a664616d2fc7df842d29aa6b5ccc9